docs/docs-beta/docs/guides/automation/schedules.md

```diff
@@ -1,57 +1,35 @@
 ---
-title: "Scheduling pipelines"
+title: "Scheduling cron-based pipelines"
 sidebar_label: "Running pipelines on a schedule"
 sidebar_position: 10
 ---
 
-## Basic Schedule Example
+Schedules enable automated execution of jobs at specified intervals. These intervals can range from common frequencies like hourly, daily, or weekly, to more intricate patterns defined using cron expressions.
```

Comment:
Same comment here about this page not following our guide structure - missing prereqs, what this guide does, etc.